concepts/Common/Data Analytics and BI/Embedded Reports/Embedded Reports.md

```diff
@@ -0,0 +1,17 @@
+Explore [DevExpress Reporting](https://www.devexpress.com/subscriptions/reporting/) - a reporting solution designed to meet your needs for advanced reporting functionality, rich design-time experience, and extensive customization options.
+
+![DevExpress Reporting](/images/Common/Embedded_Reports.png)
+
+#### Key Features
+
+- Subreports, master-detail reports, report parameters, expressions, data bindings, and conditional formatting.
+
+- An intuitive drag-and-drop interface, [integration with Visual Studio](https://docs.devexpress.com/XtraReports/4256/visual-studio-report-designer) and [VS Code](https://docs.devexpress.com/XtraReports/404722/vs-code-report-designer).
+
+- [End-User Report Designer](https://docs.devexpress.com/XtraReports/404503/get-started-with-devexpress-reporting/create-an-end-user-designer) to create and edit reports at runtime.
+
+- [Web Document Viewer](https://docs.devexpress.com/XtraReports/403569/web-reporting/common-features/devexpress-document-viewers) with an interactive preview of generated documents, allowing users to view, print, and export reports.
+
+DevExpress Reporting is compatible across various platforms including ASP.NET WebForms, ASP.NET MVC, ASP.NET Core, Blazor, and JavaScript applications based on Angular, React, and Vue frameworks.
+
+[note] You can select from a range of pricing options, including standalone Reporting subscriptions or the Universal package, which includes both DevExpress Reporting and DevExtreme.
```

Comment:
Let's make subscription mentions to be links to the corresponding pages (as in other sections)